docs/topics/mta-rn-new-features-1.adoc

```diff
@@ -8,6 +8,6 @@
 
 This section describes the new features and improvements of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.Support for non-administrators to run {ProductShortName}
-For security reasons, the RWX permission set of {ProductShortName} 6.0.0 was configured to allow only administrators to run {ProductShortName}. In Version {ProductVersion}  you can now allow non-administrators to use {ProductShortName} by setting the value of a new parameter, `rwx_supported`, to `false` in the the {ProductShortName} Operator or the {ocp-first}  console.
+.{ProductShortName} no longer requires RWX permissions to store data in PVs
```

Comment:
I think that we should use language that explains why we added this configuration option and what it's default value is.

When RWX is supported, the operator create a cache volume to be used as the local Maven repository (https://maven.apache.org/guides/introduction/introduction-to-repositories.html). When not supported, by default the maven local repository will be localized to each individual task.

I think it's important to clarify what this feature does as these current docs make it seem like we are having the user configure the PV settings for the cache.

Reply:
Sourced this from: https://issues.redhat.com/browse/MTA-44